examples/webpack.config.babel.js

```diff
@@ -0,0 +1,33 @@
+import path from 'path'
+import HtmlWebpackPlugin from 'html-webpack-plugin'
+
+export default {
+  entry: path.resolve('./examples/main.js'),
+  output: {
+    path: path.resolve('./examples/dist'),
+    filename: 'main.js',
+  },
+  plugins: [new HtmlWebpackPlugin()],
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            babelrc: false,
+            plugins: [
+              ['transform-class-properties', { loose: true }],
+              'transform-object-rest-spread',
+            ],
+            presets: [
+              ['latest', { es2015: { loose: true, modules: false } }],
```

Comment:
Apparently, the `latest` preset is [deprecated](https://babeljs.io/docs/plugins/preset-latest/).

It seems like the "new" way is `['env', { loose: true, modules: false } ]`.